js/components/frame.js

```diff
@@ -99,6 +99,7 @@ class Frame extends ImmutableComponent {
     const allowRunningInsecureContent = !!(hack && hack.allowRunningInsecureContent)
     if (didSrcChange || didLocationChange && location === 'about:certerror' || !this.webview ||
         allowRunningInsecureContent !== this.webview.allowRunningInsecureContent) {
+      this.webview.setAttribute('plugins', 'true')
```

Comment:
is there a reason this is set outside of updateWebview and only on update? Seems like this would fail to set the attribute on a window restore and probably some other cases as well